config/modules/refine.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Refine",
+  "id_name": "refine",
+  "category": "Edit",
+  "icon": "edit",
+  "description": "Histogram-based data cleaning",
+  "help_url":"modules/edit/editcells",
+  "parameters": [
+    {
+      "name": "Column",
+      "id_name": "column",
+      "type": "column"
+    },
+    {
+      "name": "Histogram",
+      "id_name": "histogram",
+      "type": "custom"
+    },
+    {
+      "name": "Edits",
+      "id_name": "edits",
+      "type": "string",
+      "visible": false
```

Comment:
you could actually just store the edits in the histogram parameter... this also an older design pattern for us :/